_sources/Functions/Variablesandparametersarelocal.rst

```diff
@@ -0,0 +1,50 @@
+..  Copyright (C)  Brad Miller, David Ranum, Jeffrey Elkner, Peter Wentworth, Allen B. Downey, Chris
+    Meyers, and Dario Mitchell.  Permission is granted to copy, distribute
+    and/or modify this document under the terms of the GNU Free Documentation
+    License, Version 1.3 or any later version published by the Free Software
+    Foundation; with Invariant Sections being Forward, Prefaces, and
+    Contributor List, no Front-Cover Texts, and no Back-Cover Texts.  A copy of
+    the license is included in the section entitled "GNU Free Documentation
+    License".
+
+.. qnum::
+   :prefix: func-7-
+   :start: 1
+
+.. index:: local variable
+   variable; local
+   lifetime
+
+Variables and parameters are local
+----------------------------------
+
+An assignment statement in a function creates a **local variable** for the variable on the left hand side of the 
+assignment operator. It is called local because this variable only exists inside the function and you cannot use it 
+outside. For example, consider again the ``square`` function:
+
+.. activecode:: ac11_7_1
+
+    def square(x):
+        y = x * x
+        return y
+
+    z = square(10)
+    print(y)
+
+
+Try running this in Codelens. If you press the 'last >>' button you will see an error message. When we try to use 
```

Comment:
It would also be useful to point out that codelens identifies the local scope when a function is invoked by putting the variables inside the blue box, so you can see they are different from the global scope.